code/__HELPERS/reagents.dm

```diff
@@ -73,6 +73,12 @@
 		GLOB.chemical_reactions_list[primary_reagent] = list()
 	GLOB.chemical_reactions_list[primary_reagent] += R
 
+/proc/unit_form_string(amount)
+	if(amount <= 1)
+		return "ligula"
+	else
+		return "ligulae"
```

Comment:
```suggestion
#define UNIT_FORM_STRING(amount) (amount = 1 ? "ligula" : "ligulae")
```
Literally UNIT_FORM_STRING(volume) or [UNIT_FORM_STRING(volume)] its no different than using a proc

But move to top of file for clarity

Reply:
d o n e